Mathlib/Algebra/Order/Group/Nat.lean

```diff
@@ -20,10 +20,10 @@ namespace Nat
 /-! ### Instances -/
 
 instance instIsOrderedAddMonoid : IsOrderedAddMonoid ℕ where
-  add_le_add_left := @Nat.add_le_add_left
+  add_le_add_left := @Nat.add_le_add_right
```

Comment:
It is annoying that we'll be moving out of sync with core Lean in this naming convention. Are you planning to add a PR to upstream?